### ISSUE TITLE
Youtube Changes/Upgrades

### ISSUE THREAD

**Issue by CChavinha** [okTurtles/group-income#996]
### **Problem**

Our Youtube channel is looking outdated, it needs a few adjustments (image & keywords), and it's not cohesive with the image we're currently implementing.

- **The channel Image is cropped on preview**
![GI-Channel-Image](https://user-images.githubusercontent.com/66070659/93919350-78b4cc80-fd05-11ea-9940-0953ec876794.png)

- **The cover images isn't appealing/cohesive** 
![GI-Cover-Image](https://user-images.githubusercontent.com/66070659/93919369-7d798080-fd05-11ea-8487-1e56719b56ac.png)

- **The keywords aren't implemented. When searching for GI Shorts, the videos are nowhere to be found**
![GI-Shorts-Search](https://user-images.githubusercontent.com/66070659/93919669-e9f47f80-fd05-11ea-96f8-7293d8ebcb35.png)

### **Solution**

- **Replace channel Image with this one:**
![group-income-social-media-size-logo](https://user-images.githubusercontent.com/66070659/94006201-65ead800-fd97-11ea-9a78-edc5110f15bd.jpg)

Replace cover image with this one:
(similar to our new social-share card/let me know if you like it)
 
![GI-Youtube-Cover-Image](https://user-images.githubusercontent.com/66070659/94006469-cb3ec900-fd97-11ea-8061-bfd09182949e.jpg)

(keep the twitter call to action already in place)

**UPGRADE KEYWORDS**
Edit the GI playlist name:
![GI-Shorts-Playlist](https://user-images.githubusercontent.com/66070659/94005760-a72eb800-fd96-11ea-9ee1-0bc80879c17e.png)
“Group Income Shorts” instead of “Shorts"

Edit all video titles:
![GI-Shorts-Videos](https://user-images.githubusercontent.com/66070659/94005843-cdecee80-fd96-11ea-9ddd-9c0c11a1da42.png)

Group Income Shorts | A New Kind Of Software Architecture
Group Income Shorts | What is Value?
Group Income Shorts | What is Group Income? (Overview)

**Comment by dotmacro** [comment]
TODO list:

- [x] update GI channel image so it isn't cropped 
- [x] update GI banner image
- [x] change Group Income Shorts playlist title to "Group Income Shorts"
- [x] prepend titles of all videos in shorts playlist with "Group Income Shorts | "
- [x] double-check/update keywords of GI videos
- [x] double-check/update keywords of GI channel
- [x] claim [custom URL for GI channel](http://www.youtube.com/c/GroupIncome)
- [x] claim [custom URL for okTurtles channel](http://www.youtube.com/c/okTurtles)
- [ ] update GI video descriptions
- [ ] double-check/fix GI video subtitles
- [ ] double-check/fix okT video subtitles (a bit OT for this repo, but 🤷‍♀️)

As of when I originally posted this comment, "What is Value" was the 6th result when I search for "Group Income Shorts", even though the video title change still hasn't propagated through their system.

**Comment by F1LT3R** [comment]
I pulled the video info just now in JSON format:

```json
[
    {
      "videoId": "I381R5_joLE",
      "title": "Group Income Shorts | A New Kind Of Software Architecture",
      "lengthSeconds": "117",
      "keywords": [
        "basic income",
        "group income",
        "basicincome",
        "decentralization",
        "decentralized",
        "ethereum",
        "blockchain",
        "UBI"
      ],
      "shortDescription": "Your data belongs to you. We are not entitled to it.",
      "averageRating": 5,
      "viewCount": "542",
      "author": "Group Income"
    },

    {
      "videoId": "KU9NGcrjXlo",
      "title": "Group Income Shorts | Group Income UI Demo",
      "lengthSeconds": "490",
      "keywords": [
        "basic income",
        "group income",
        "decentralization",
        "software",
        "financial",
        "UBI",
        "basicincome"
      ],
      "shortDescription": "A deceptively simple demo! https://groupincome.org/2020/05/a-deceptively-simple-demo/",
      "averageRating": 5,
      "viewCount": "152",
      "author": "Group Income"
    },


    {
      "videoId": "WpvyLEZmEAM",
      "title": "Group Income Shorts | What is Group Income? (Overview)",
      "lengthSeconds": "73",
      "keywords": [
        "basic income",
        "basicincome",
        "group income",
        "decentralization",
        "voluntaryism",
        "UBI"
      ],
      "shortDescription": "The first short in our new series of explainer videos about Group Income. Visit https://groupincome.org for more info!",
      "averageRating": 5,
      "viewCount": "1329",
      "author": "Group Income"
    },

    {
      "videoId": "i3LtljP9WsA",
      "title": "Group Income Shorts | What is Value?",
      "lengthSeconds": "116",
      "keywords": [
        "group income",
        "basic income",
        "value",
        "monetize",
        "monetization",
        "open source",
        "women",
        "economics",
        "UBI"
      ],
      "shortDescription": "",
      "averageRating": 4.4545455,
      "viewCount": "30351",
      "author": "Group Income"
    }
]
``` 

Forgive me if this is out-of-scope for this PR, but I would think the following may be worth discussing at some point:

1. **Update Descriptions** - Add a short summary of the spoken content each video. This should help with SEO. 
2. **Adding Keywords**
    1. We may want more keywords.
    2. It could be worth calling out "Universal Basic Income" as it's own keyword in each video, as this is becoming such a hot-topic in the media.
    3. One SEO trick is to combine two words as one phrase and use this as a keyword, eg: "GroupIncome". This makes things much easier to find (especially if you mirror this in the marks/logos), as you are essentially inventing your own word.

**Comment by dotmacro** [comment]
> It could be worth calling out "Universal Basic Income" as it's own keyword
> use this as a keyword, eg: "GroupIncome"

Good call. I've added "Universal Basic Income" and "GroupIncome" to the video and channel keywords.

**Comment by taoeffect** [comment]
Catarina, regarding the banner and "OkTurtles! Foundation" <- the O should be lowercase (okTurtles) and there's no exclamation mark. I like the new banner, however, the sentence in the upper right does not look good to me and I think it should simply be removed, or made smaller and lighter font and placed in the lower-right. If it's used, please use the [ProximaNova Light](https://github.com/okTurtles/okturtles.com/tree/master/includes/fonts) font so that it matches the font on our website.

**Comment by CChavinha** [comment]
Yes, I liked it with the okTurtles on top, but I should have sent other versions so that we could agree on the best one.
Here it is with okTurtles written correctly and with [ProximaNova](https://github.com/okTurtles/okturtles.com/tree/master/includes/fonts). 
Let me know what you think:

![GI-Youtube-Cover-Image-Version1](https://user-images.githubusercontent.com/66070659/95228673-8b47ff00-07f7-11eb-8edd-314c7b356e7b.jpg)
![GI-Youtube-Cover-Image-Version2](https://user-images.githubusercontent.com/66070659/95228717-9733c100-07f7-11eb-8546-e3578720d33d.jpg)
![GI-Youtube-Cover-Image-Version3](https://user-images.githubusercontent.com/66070659/95228736-9c910b80-07f7-11eb-87c8-ded3d72ba6aa.jpg)

**Comment by sdegutis** [comment]
If I can vote: I like the middle one the best. Clever idea using the sentence to lead up to the "g".

**Comment by CChavinha** [comment]
@taoeffect did you save a written version of what you say on the videos?
It would be helpful for the new descriptions

**Comment by CChavinha** [comment]
> If I can vote: I like the middle one the best. Clever idea using the sentence to lead up to the "g".

It's my favorite one as well!

**Comment by taoeffect** [comment]
@CChavinha Thank you! These are looking much better! Can you modify the middle and bottom versions to remove the okTurtles logo? I feel that it stands out way too much and we need only one logo to be visible in this banner, and that is the Group Income logo.

**Comment by CChavinha** [comment]
Sure!
![GI-Youtube-Cover-Image-Version5](https://user-images.githubusercontent.com/66070659/95231295-e0394480-07fa-11eb-853b-451e5ebd0adb.jpg)
![GI-Youtube-Cover-Image-Version4](https://user-images.githubusercontent.com/66070659/95231288-ddd6ea80-07fa-11eb-851a-d40dfe7f3dc0.jpg)

You're right, looks very clean like this! The okTurtles logo was cool, but too much.

**Comment by dotmacro** [comment]
Excellent! I think the first one (most similar to previous middle) looks the cleanest.

**Comment by CChavinha** [comment]
Agreed!

**Comment by taoeffect** [comment]
Alright! Let's go with the first one in [this post](https://github.com/okTurtles/group-income-simple/issues/996#issuecomment-704404006) above, fantastic work @CChavinha! 😄
